packages/grafana-data/src/field/displayProcessor.ts

```diff
@@ -77,7 +77,7 @@ export function getDisplayProcessor(options?: DisplayProcessorOptions): DisplayP
     const isStringUnit = unit === 'string';
 
     if (hasDateUnit && typeof value === 'string') {
-      value = dateTime(value).valueOf();
+      value = toUtc(value).valueOf();
```

Comment:
Think we should default to UTC only if the display processor is not configured with a timezone (it can be passed via display processor's options).

Reply:
ah wait, ignore the comment above. We are talking date strings here. Thanks @oscarkilhed for the tests, this made it slightly clearer.